polls/admin.py

```diff
@@ -30,7 +33,34 @@ def queryset(self, request, queryset):
         return queryset
 
 
+def download_as_csv(QuestionAdmin, request, queryset):
```

Comment:
Could we lowercase `QuestionAdmin` since it is an instance? Sorry, not-so-useful bikeshed.